tools/harness-application

```diff
@@ -54,8 +55,12 @@ if __name__ == "__main__":
 
     app_path = os.path.join(APPLICATIONS_SRC_PATH, args.name)
     for template_name in args.templates:
+
         if template_name == 'server':
-            generate_server(app_path)
+            with tempfile.TemporaryDirectory() as tmp_dirname:
+                copymergedir(os.path.join(HERE, APPLICATION_TEMPLATE_PATH, template_name), tmp_dirname)
+                merge_configuration_directories(app_path, tmp_dirname)
+                generate_server(app_path, tmp_dirname)
```

Comment:
Have to remove the tmp path at the end (suggest to use try...finally to ensure it is deleted)

Reply:
This should be happening automatically because I'm creating the temp folder using the context manager *with*

Reply:
right, nice!